src/messages.cs

```diff
@@ -203,7 +203,7 @@ interface MSMessageTemplateLayout
 
 	[iOS (10,0)]
 	[BaseType (typeof(NSObject))]
-	interface MSSession {}
+	interface MSSession : NSSecureCoding {}
```

Comment:
Try running this on older sims - it will likely need an adjustment to the intro tests